src/networks.json

```diff
@@ -2111,21 +2111,21 @@
     "start": 43317,
     "logo": "ipfs://QmXvum7SNVaAqAc2jNzR1NpNZN1GGvNaKWydg8a1GEDQ7y"
   },
-  "9000": {
-    "key": "9000",
+  "6102": {
+    "key": "6102",
     "name": "Cascadia Testnet",
     "shortName": "Cascadia",
-    "chainId": 9000,
+    "chainId": 6102,
     "network": "testnet",
-    "multicall": "0xBd186aca63bD4f5f4eC9E98028461A6A94ac4a12",
+    "multicall": "0x728989819bAD588F193563008E0a03E8cD6a3e4a",
```

Comment:
Hi @recursivethedev Can you plz verify and publish code here https://explorer.cascadia.foundation/address/0x728989819bAD588F193563008E0a03E8cD6a3e4a/contracts#address-tabs

Reply:
Hi @ChaituVR , We have verified and publish the contract. Can you please review if everything is good ?